{"uuid": "09e2c357-2267-4a1c-8fc3-57625a94685a", "children": ["55a23e2d-a514-4727-9146-4fecbb7dfb7e", "316df108-3faf-41b5-85bd-71b7702b5df5"], "befores": [{"name": "record", "status": "passed", "start": 1592383425086, "stop": 1592383425111}], "afters": [{"name": "record::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'CompletedProcess' object has no attribute 'pid'\n", "trace": "  File \"c:\\testtools\\python\\python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\testtools\\python\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\heli\\PycharmProjects\\diy_appium\\scripts\\conftest.py\", line 21, in record\n    os.kill(p.pid, signal.CTRL_C_EVENT)\n"}, "start": 1592383464034, "stop": 1592383464035}], "start": 1592383425086, "stop": 1592383464035}